internal/jobservice/configuration/types.go

```diff
@@ -0,0 +1,20 @@
+package configuration
+
+import (
+	"github.com/go-redis/redis"
+
+	grpcconfig "github.com/G-Research/armada/internal/common/grpc/configuration"
+	"github.com/G-Research/armada/pkg/client"
+)
+
+type JobServiceConfiguration struct {
```

Comment:
I think we can re-use the config we already use for the Armada client? The test suite also takes this approach.

Reply:
Nice find!  I attempted to do this.  Please take a look.